### PR TITLE
Partial content

### PR DESCRIPTION
**Pull request by hkgumbs**
This PR passes ResponseTestSuite.PartialContent
@trptcolin @robertjlooby

### CODE REVIEW

File: test/cob_app/get_test.clj [hunk 1]
```diff
@@ -19,9 +19,9 @@
     (.delete (io/file "./tmp/base64_image")))
 
   (after-all
-    (.delete (io/file "./tmp/file"))
-    (.delete (io/file "./tmp/image.gif"))
-    (.delete (io/file "./tmp")))
+    (io/delete-file "./tmp/file")
+    (io/delete-file "./tmp/image.gif")
+    (io/delete-file "./tmp"))
```
Comment (robertjlooby) [inline test/cob_app/get_test.clj]: I like this change. It's usually nicer to make cleanup/refactor changes like this (that are unrelated to the commit) in a separate commit though just to keep each commit to one coherent action/make it clearer when reviewing.

**Comment by robertjlooby** [comment]
:+1:

File: test/cob_app/get_test.clj (hunk 1 above)
Comment (hkgumbs) [inline test/cob_app/get_test.clj]: That makes sense. Will keep in mind for future
